assets/js/params/File/UploadApiModal.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types'
 import { Modal, ModalHeader, ModalBody, ModalFooter } from '../../components/Modal'
```

Comment:
I am not sure, but I believe we might as well translate it for consistency. 

Could a Greek journalist with little knowledge on programming find it more useful if it were translated? I can't tell; maybe we should ask Catherine for input on this?

Reply:
By the way, how can I see this modal?

Reply:
How many Greek programmers don't understand English? I'm assuming non-English programmers learn enough English to get by.... (Looking around, I see https://developers.facebook.com/docs/live-video-api -- Facebook API docs are translated into 17 languages, while Facebook itself is translated into 100 languages.)

Do you feel a non-technical translator will accurately transform this English into Greek (or another language)? Personally, I can't see myself asking translators to translate technical documentation. ... Then again, my goal here is to avoid wasting time; so I'll merge and then we can revisit later.

The modal is behind a "feature flag" because we haven't finished designing it. Open a Workflow with an "Upload" module; then add `#feature:uploadapi` to the URL and refresh the page. Click the new "API" button (beside the "Replace" button).